src/editor/EditorCommandHandlers.js

```diff
@@ -45,7 +43,7 @@ define(function (require, exports, module) {
     var commandKey = {
         COMMAND_KEY: "Ctrl"
     };
-    if(true){
+    if(navigator.platform.indexOf("Mac") !== -1;){
         commandKey = {
             COMMAND_KEY: "&#8984"
```

Comment:
@humphd This was the code I mentioned in the email. Where the Unicode doesn't display the Command symbol.